### ISSUE TITLE
DataExporter : column headers are not being exported to XLS, XLSX, PDF

### ISSUE THREAD

**Issue by tarwork** [primefaces/primefaces#9290]
### Describe the bug

using the current PrimeFaces showcase located here :

https://www.primefaces.org/showcase/ui/data/dataexporter/basic.xhtml?jfwid=4b24b

when clicking on the XLS, XLSX and PDF buttons, the table column headers seem to be missing.

Primefaces 11 seem to have them so this must be a new issue to PF12.

### Reproducer

https://www.primefaces.org/showcase/ui/data/dataexporter/basic.xhtml?jfwid=4b24b

### Expected behavior

Expected behavior is that the column headers make it to the XLS, XLSX and PDF export.

### PrimeFaces edition

Community

### PrimeFaces version

12.0.0

### Theme

default for PF Showcase

### JSF implementation

Mojarra

### JSF version

unknown

### Java version

unknown

### Browser(s)

fails in Chrome and Firefox

**Comment by melloware** [comment]
Yep looks like i broke it with this fix: https://github.com/primefaces/primefaces/issues/9179

**Comment by melloware** [comment]
Sorry about that I asked for it to be put in 12.0.1 Elite.

**Comment by melloware** [comment]
Repinning this issue so people can see it up top until 12.0.1 is released so people don't constantly re-report it.

**Comment by SirError** [comment]
Hi @melloware , is there a way to override this class on my app so I could apply this fix?

I usually do something like

```
	<component>
	    <component-type>org.primefaces.component.DataTable</component-type>
	    <component-class>.com.project.CustomDataTable</component-class>
	</component>
```

but DataTableExcelExporter doesn't have a COMPONENT_TYPE attribute.

**Comment by melloware** [comment]
Not that I am aware of maybe @tandraschko knows but if you are an Elite subscriber this has already been fixed and published in 12.0.1.

**Comment by fabrii** [comment]
Hi @melloware and @tandraschko. As this seems to be a regression bug, is there any chance to release a 12.0.1 version to the community with this fix?

Thank you!

**Comment by melloware** [comment]
That is up to PrimeTek and they typically have never done that in the past.  Most likely you will need to wait for 13.0.0 or if you are an Elite subscriber you could download the fix right now.

**Comment by buddhika75** [comment]
It's fixed in 14.0.0

**Comment by melloware** [comment]
@buddhika75 its actually fixed in 13.0.0-13.0.10 as well.
